practical-penguins/trivia_tavern/twilio_messenger/models.py

```diff
@@ -0,0 +1,43 @@
+from typing import Tuple, List, Optional
+
+from django.db import models
+from trivia_runner.models import Player
+
+
+class ScoreTracker(models.Model):
+    player_phone = models.CharField(max_length=12)
+    team_name = models.CharField(max_length=24, default='')
+    points = models.IntegerField(default=0)
+    session_code = models.CharField(max_length=6)
+    answered_this_round = models.BooleanField(default=False)
+
+    @staticmethod
+    def get_score_list(session_code: int) -> List[Optional[Tuple[str, int]]]:
+        """get_score_list is a class method that returns all players and their score_list
+        for the specified session"""
+        return [(x.player_phone, x.points) for x in ScoreTracker.objects.filter(session_code=session_code)]
+
+    @staticmethod
+    def get_team_score_list(session_code: int) -> List[Optional[Tuple[int, str, int]]]:
+        """As above, but for each team"""
+        # First get all possible team names
```

Comment:
maybe not needed to inline comment something that is obvious. your list-comprehension is clear enough on its own

Reply:
It's also not a great practice to use "As above" in the docstring either, as one major reason to use docstrings in general is that when you either view the `__doc__` of the function manually or through a tooltip in an IDE, it shows a description of what it does without having to refer to the function directly. So, "above" means nothing in that context. It would be better to give it a similar description or specifically refer to the similar function, `get_score_list()`.